Windows.Clang.toolchain.cmake

```diff
@@ -55,7 +59,7 @@ if(NOT (CMAKE_HOST_SYSTEM_NAME STREQUAL Windows))
     return()
 endif()
 
-set(UNUSED ${CMAKE_TOOLCHAIN_FILE})
+set(UNUSED ${CMAKE_TOOLCHAIN_FILE}) # TODO(CK): what does this mean?
```

Comment:
It 'uses' the `CMAKE_TOOLCHAIN_FILE` variable to avoid warnings discussed on this StackOverflow thread: https://stackoverflow.com/questions/14757506/why-am-i-warned-that-the-cmake-toolchain-file-variable-is-not-used-by-the-proj